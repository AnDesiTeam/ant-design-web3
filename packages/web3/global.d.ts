declare const YOUR_ZAN_API_KEY: string;
declare const YOUR_WALLET_CONNET_PROJECT_ID: string;
declare const YOUR_INFURA_API_KEY: string;
