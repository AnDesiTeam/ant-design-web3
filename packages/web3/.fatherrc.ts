import { defineConfig } from 'father';

export default defineConfig({
  extends: '../../.fatherrc.base.ts',
  extraBabelPlugins: ['inline-react-svg'],
});
