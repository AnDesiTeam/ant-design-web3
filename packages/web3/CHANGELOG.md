# @ant-design/web3

## 0.3.0

### Minor Changes

- 80b1957: feat: ConnectModal support mode
- a2119bf: refactor: rename hasBrowserExtensionInstalled to hasWalletReady and rename onSelectWallet to onWalletSelected

### Patch Changes

- a86e7c6: fix: qrcode style bug in simple ConnectModal
- caa86c4: fix: Get wallet dowload bug
- bf23a50: fix: ConnectModal info style bug
- d319842: fix: use antd Modal design token
- 42b7ebe: fix: ConnectModal button height, use antd large button
- e9f8fd4: fix: Get wallet button not work in ConnectModal panel mode
- 7bae140: fix: ConnectButton text color bug
- ab50e5a: fix: ConnectModal QrCode link style
- 5dbecd5: fix: NFTCard action button click not work
- Updated dependencies [a2119bf]
  - @ant-design/web3-common@0.2.0

## 0.2.0

### Minor Changes

- 606136b: Avatar is displayed in connect button component

### Patch Changes

- c57eed3: feat: export ConnectModal.ModalPanel for user custom
- 2d7ae4f: refactor: auto config wagmi provider chains
- 0df085f: feat: Add and update circle icons
- 1639b80: feat: connect modal build-in guide
- cb5d1eb: fix: Should not show Disconnect and Copy Address action in dropdown menu when not connected
- d3e5988: feat: add link for wallet qrcode for support connect in mobile
- 96837b3: fix: NFTCard ETH icon style detail
- a2d17cd: feat: optimize connect-modal close icon style
- c508982: feat: optimize connect modal style
- 3fbd1eb: fix: Connect trigger onDisconnectClick type bug
- e6a30be: feat: browser link support get chain icon from provider
- 1d2bb80: feat: New hook useConnction ‚≠ê
- 39782e4: feat: ethereum adapter support show balance
- 78b9282: feat: New component CryptoPrice üöÄ
- fe42780: fix: chain icon size in avatar
- c15bfdf: feat: BrowserLink support get url from chain in @ant-design/web3-assets
- ee9754a: feat: NFTCard show Skeleton when request metadata
- bf5ec8e: feat: ConnectButton support balance
- Updated dependencies [2d7ae4f]
- Updated dependencies [0df085f]
- Updated dependencies [9bac342]
- Updated dependencies [3fbd1eb]
- Updated dependencies [39782e4]
- Updated dependencies [c15bfdf]
  - @ant-design/web3-common@0.1.1
  - @ant-design/web3-icons@0.2.0

## 0.1.0

### Patch Changes

- Release new alpha version
- Updated dependencies
  - @ant-design/web3-common@0.1.0
  - @ant-design/web3-icons@0.1.0

## 0.1.0-alpha.1

### Patch Changes

- First alpha version.
- Updated dependencies
  - @ant-design/web3-common@0.1.0-alpha.1
  - @ant-design/web3-icons@0.1.0-alpha.1
