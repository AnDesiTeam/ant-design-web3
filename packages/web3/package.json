{
  "name": "@ant-design/web3",
  "version": "1.0.0",
  "exports": {
    "import": "./dist/esm/index.js",
    "require": "./dist/lib/index.js"
  },
  "files": [
    "dist",
    "CHANGELOG.md",
    "README.md"
  ],
  "scripts": {
    "dev": "father dev",
    "build": "father build"
  },
  "dependencies": {
    "@ant-design/cssinjs": "^1.17.0",
    "@ant-design/icons": "^5.2.6",
    "@ant-design/web3-icons": "workspace:*",
    "@ctrl/tinycolor": "^4.0.2",
    "antd": "^5.8.3",
    "classnames": "^2.3.2"
  },
  "devDependencies": {
    "@ant-design/web3-adapter-ethers": "workspace:*",
    "@types/react": "^18.2.20",
    "@types/react-dom": "^18.2.7",
    "father": "^4.1.9",
    "typescript": "^5.0.4"
  },
  "peerDependencies": {
    "antd": ">=4.23.0 || >=5.0.0",
    "react": ">=17.0.0",
    "react-dom": ">=17.0.0"
  },
  "publishConfig": {
    "registry": "https://registry.npmjs.org",
    "access": "public"
  }
}
