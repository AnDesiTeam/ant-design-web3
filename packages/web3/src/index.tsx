export * from '@ant-design/web3-common';
export * from './ConnectButton';
export * from './ConnectModal';
export * from './Web3ConfigProvider';
export * from './MockProvider';
export * from './Address';
export * from './BrowserLink';
export * from './NFTCard';
export * from './constants';
