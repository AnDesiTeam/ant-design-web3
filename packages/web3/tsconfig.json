{
  "extends": "../../tsconfig.base.json",
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@ant-design/web3": ["./src/index.tsx"],
      "@ant-design/web3-icons": ["../icons/src/index.tsx"]
    }
  },
  "include": ["src", "global.d.ts"]
}
