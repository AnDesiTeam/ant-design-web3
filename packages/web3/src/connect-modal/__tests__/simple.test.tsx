import { ConnectModal } from '@ant-design/web3';
import { groupOrder, walletList } from './mock';
import { render } from '@testing-library/react';
import { it, describe, expect } from 'vitest';

describe('ConnectModal without guide', () => {
  it.each(['light', 'dark'] as const)(`should render in %s mode`, async (theme) => {
    const App = () => (
      <ConnectModal
        open
        theme={theme}
        title="ConnectModal"
        footer="蚂蚁链提供技术支持"
        groupOrder={groupOrder}
        walletList={walletList}
      />
    );
    const { baseElement } = render(<App />);
    expect(baseElement).toMatchSnapshot();
    // should have ant-connect-modal class
    expect(baseElement.querySelector('.ant-modal')?.className).toContain('ant-connect-modal');
    expect(baseElement.querySelector('.ant-modal')?.className).toContain(
      `ant-connect-modal-${theme}`,
    );

    // should have simple class when without guide
    expect(baseElement.querySelector('.ant-connect-modal-body')?.className).toContain('simple');

    // should have title and footer
    expect(baseElement.querySelector('.ant-connect-modal-title')?.textContent).toBe('ConnectModal');
    expect(baseElement.querySelector('.ant-connect-modal-footer')?.textContent).toBe(
      '蚂蚁链提供技术支持',
    );

    // group order
    expect(baseElement.querySelectorAll('.ant-connect-modal-group-title')[0].textContent).toBe(
      'Popular',
    );
    expect(baseElement.querySelectorAll('.ant-connect-modal-group-title')[1].textContent).toBe(
      'Default',
    );

    // wallet list items
    expect(baseElement.querySelectorAll('.ant-connect-modal-wallet-item').length).toBe(
      walletList.length,
    );
  });
});
