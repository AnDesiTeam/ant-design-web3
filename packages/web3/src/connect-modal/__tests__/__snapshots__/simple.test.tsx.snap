// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ConnectModal without guide > should render in dark mode 1`] = `
<body>
  <div />
  <div>
    <div
      class="ant-modal-root css-dev-only-do-not-override-15xztg8 ant-web3-connect-modal-root"
    >
      <div
        class="ant-modal-mask ant-fade-appear ant-fade-appear-start ant-fade"
      />
      <div
        class="ant-modal-wrap"
        tabindex="-1"
      >
        <div
          aria-modal="true"
          class="ant-modal css-dev-only-do-not-override-15xztg8 ant-web3-connect-modal css-dev-only-do-not-override-15xztg8 ant-zoom-appear ant-zoom-appear-prepare ant-zoom"
          role="dialog"
          style="width: 380px;"
        >
          <div
            aria-hidden="true"
            style="width: 0px; height: 0px; overflow: hidden; outline: none;"
            tabindex="0"
          />
          <div
            class="ant-modal-content"
          >
            <button
              aria-label="Close"
              class="ant-modal-close"
              type="button"
            >
              <span
                class="ant-modal-close-x"
              >
                <span
                  aria-label="close-circle"
                  class="anticon anticon-close-circle"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="close-circle"
                    fill="currentColor"
                    fill-rule="evenodd"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M512 64c247.4 0 448 200.6 448 448S759.4 960 512 960 64 759.4 64 512 264.6 64 512 64zm127.98 274.82h-.04l-.08.06L512 466.75 384.14 338.88c-.04-.05-.06-.06-.08-.06a.12.12 0 00-.07 0c-.03 0-.05.01-.09.05l-45.02 45.02a.2.2 0 00-.05.09.12.12 0 000 .07v.02a.27.27 0 00.06.06L466.75 512 338.88 639.86c-.05.04-.06.06-.06.08a.12.12 0 000 .07c0 .03.01.05.05.09l45.02 45.02a.2.2 0 00.09.05.12.12 0 00.07 0c.02 0 .04-.01.08-.05L512 557.25l127.86 127.87c.04.04.06.05.08.05a.12.12 0 00.07 0c.03 0 .05-.01.09-.05l45.02-45.02a.2.2 0 00.05-.09.12.12 0 000-.07v-.02a.27.27 0 00-.05-.06L557.25 512l127.87-127.86c.04-.04.05-.06.05-.08a.12.12 0 000-.07c0-.03-.01-.05-.05-.09l-45.02-45.02a.2.2 0 00-.09-.05.12.12 0 00-.07 0z"
                    />
                  </svg>
                </span>
              </span>
            </button>
            <div
              class="ant-modal-body"
            >
              <div
                class="ant-web3-connect-modal-body simple"
              >
                <div
                  class="ant-web3-connect-modal-list-panel"
                >
                  <div
                    class="ant-web3-connect-modal-header"
                  >
                    <h2
                      class="ant-web3-connect-modal-title"
                    >
                      ConnectModal
                    </h2>
                  </div>
                  <div
                    class="ant-web3-connect-modal-list-container"
                  >
                    <div
                      class="ant-web3-connect-modal-wallet-list"
                    >
                      <div
                        class="ant-web3-connect-modal-group"
                      >
                        <div
                          class="ant-web3-connect-modal-group-title"
                        >
                          Popular
                        </div>
                        <div
                          class="ant-web3-connect-modal-group-content"
                        >
                          <div
                            class="ant-list ant-list-split css-dev-only-do-not-override-15xztg8"
                          >
                            <div
                              class="ant-spin-nested-loading css-dev-only-do-not-override-15xztg8"
                            >
                              <div
                                class="ant-spin-container"
                              >
                                <ul
                                  class="ant-list-items"
                                >
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包"
                                          src="https://xsgames.co/randomusers/avatar.php?g=pixel&key=0"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包3"
                                          src="https://xsgames.co/randomusers/avatar.php?g=pixel&key=3"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包3
                                      </div>
                                    </div>
                                  </li>
                                </ul>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        class="ant-web3-connect-modal-group"
                      >
                        <div
                          class="ant-web3-connect-modal-group-title"
                        >
                          More
                        </div>
                        <div
                          class="ant-web3-connect-modal-group-content"
                        >
                          <div
                            class="ant-list ant-list-split css-dev-only-do-not-override-15xztg8"
                          >
                            <div
                              class="ant-spin-nested-loading css-dev-only-do-not-override-15xztg8"
                            >
                              <div
                                class="ant-spin-container"
                              >
                                <ul
                                  class="ant-list-items"
                                >
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包2"
                                          src="https://xsgames.co/randomusers/avatar.php?g=pixel&key=1"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包2
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包4"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包4
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包5"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包5
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包6"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包6
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包6"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包6
                                      </div>
                                    </div>
                                  </li>
                                </ul>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="ant-web3-connect-modal-footer"
                  >
                    蚂蚁链提供技术支持
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            style="width: 0px; height: 0px; overflow: hidden; outline: none;"
            tabindex="0"
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;

exports[`ConnectModal without guide > should render in light mode 1`] = `
<body>
  <div />
  <div>
    <div
      class="ant-modal-root css-dev-only-do-not-override-1ck3jst ant-web3-connect-modal-root"
    >
      <div
        class="ant-modal-mask ant-fade-appear ant-fade-appear-start ant-fade"
      />
      <div
        class="ant-modal-wrap"
        tabindex="-1"
      >
        <div
          aria-modal="true"
          class="ant-modal css-dev-only-do-not-override-1ck3jst ant-web3-connect-modal css-dev-only-do-not-override-1ck3jst ant-zoom-appear ant-zoom-appear-prepare ant-zoom"
          role="dialog"
          style="width: 380px;"
        >
          <div
            aria-hidden="true"
            style="width: 0px; height: 0px; overflow: hidden; outline: none;"
            tabindex="0"
          />
          <div
            class="ant-modal-content"
          >
            <button
              aria-label="Close"
              class="ant-modal-close"
              type="button"
            >
              <span
                class="ant-modal-close-x"
              >
                <span
                  aria-label="close-circle"
                  class="anticon anticon-close-circle"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="close-circle"
                    fill="currentColor"
                    fill-rule="evenodd"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M512 64c247.4 0 448 200.6 448 448S759.4 960 512 960 64 759.4 64 512 264.6 64 512 64zm127.98 274.82h-.04l-.08.06L512 466.75 384.14 338.88c-.04-.05-.06-.06-.08-.06a.12.12 0 00-.07 0c-.03 0-.05.01-.09.05l-45.02 45.02a.2.2 0 00-.05.09.12.12 0 000 .07v.02a.27.27 0 00.06.06L466.75 512 338.88 639.86c-.05.04-.06.06-.06.08a.12.12 0 000 .07c0 .03.01.05.05.09l45.02 45.02a.2.2 0 00.09.05.12.12 0 00.07 0c.02 0 .04-.01.08-.05L512 557.25l127.86 127.87c.04.04.06.05.08.05a.12.12 0 00.07 0c.03 0 .05-.01.09-.05l45.02-45.02a.2.2 0 00.05-.09.12.12 0 000-.07v-.02a.27.27 0 00-.05-.06L557.25 512l127.87-127.86c.04-.04.05-.06.05-.08a.12.12 0 000-.07c0-.03-.01-.05-.05-.09l-45.02-45.02a.2.2 0 00-.09-.05.12.12 0 00-.07 0z"
                    />
                  </svg>
                </span>
              </span>
            </button>
            <div
              class="ant-modal-body"
            >
              <div
                class="ant-web3-connect-modal-body simple"
              >
                <div
                  class="ant-web3-connect-modal-list-panel"
                >
                  <div
                    class="ant-web3-connect-modal-header"
                  >
                    <h2
                      class="ant-web3-connect-modal-title"
                    >
                      ConnectModal
                    </h2>
                  </div>
                  <div
                    class="ant-web3-connect-modal-list-container"
                  >
                    <div
                      class="ant-web3-connect-modal-wallet-list"
                    >
                      <div
                        class="ant-web3-connect-modal-group"
                      >
                        <div
                          class="ant-web3-connect-modal-group-title"
                        >
                          Popular
                        </div>
                        <div
                          class="ant-web3-connect-modal-group-content"
                        >
                          <div
                            class="ant-list ant-list-split css-dev-only-do-not-override-1ck3jst"
                          >
                            <div
                              class="ant-spin-nested-loading css-dev-only-do-not-override-1ck3jst"
                            >
                              <div
                                class="ant-spin-container"
                              >
                                <ul
                                  class="ant-list-items"
                                >
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包"
                                          src="https://xsgames.co/randomusers/avatar.php?g=pixel&key=0"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包3"
                                          src="https://xsgames.co/randomusers/avatar.php?g=pixel&key=3"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包3
                                      </div>
                                    </div>
                                  </li>
                                </ul>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        class="ant-web3-connect-modal-group"
                      >
                        <div
                          class="ant-web3-connect-modal-group-title"
                        >
                          More
                        </div>
                        <div
                          class="ant-web3-connect-modal-group-content"
                        >
                          <div
                            class="ant-list ant-list-split css-dev-only-do-not-override-1ck3jst"
                          >
                            <div
                              class="ant-spin-nested-loading css-dev-only-do-not-override-1ck3jst"
                            >
                              <div
                                class="ant-spin-container"
                              >
                                <ul
                                  class="ant-list-items"
                                >
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包2"
                                          src="https://xsgames.co/randomusers/avatar.php?g=pixel&key=1"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包2
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包4"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包4
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包5"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包5
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包6"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包6
                                      </div>
                                    </div>
                                  </li>
                                  <li
                                    class="ant-list-item ant-web3-connect-modal-wallet-item"
                                  >
                                    <div
                                      class="ant-web3-connect-modal-content"
                                    >
                                      <div
                                        class="ant-web3-connect-modal-icon"
                                      >
                                        <img
                                          alt="测试钱包6"
                                        />
                                      </div>
                                      <div
                                        class="ant-web3-connect-modal-name"
                                      >
                                        测试钱包6
                                      </div>
                                    </div>
                                  </li>
                                </ul>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="ant-web3-connect-modal-footer"
                  >
                    蚂蚁链提供技术支持
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            style="width: 0px; height: 0px; overflow: hidden; outline: none;"
            tabindex="0"
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;
